### ISSUE TITLE
Easy mode doesn't work on individual files

### ISSUE THREAD

**Issue by 0n-s** [complexlogic/loudgain#137]
A handful of music comes as a single FLAC file. It's not part of any album. & considering it's only one track, it seems like boilerplate to make a directory for it.

I request that easy mode be able to work with these files & just assume there is no album at play given that a specific file was given as an argument.

**Comment by gsauthof** [comment]
Hm, you mean something like

```
rsgain easy somefile.m4a
```

should work as a shortcut to

```
rsgain custom -s i -l -18 -c p -m 0.0 somefile.m4a
```

or even something like

```
mkdir tmp
ln -s ../somefile.m4a tmp/
rsgain easy -p no_album tmp
rm -rf tmp
```

right?

I can see the value of such a change, even if it 'just' fixes a paper cut, arguably.

**Comment by 0n-s** [comment]
> Hm, you mean something like
> 
> ```
> rsgain easy somefile.m4a
> ```
> 
> should work as a shortcut to
> 
> ```
> rsgain custom -s i -l -18 -c p -m 0.0 somefile.m4a
> ```

Yes, although the main convenience offered by this is the ability to use the preset files with single files.
